python/chemiscope/explore/__init__.py

```diff
@@ -1,22 +1,73 @@
+import warnings
+
 from ..jupyter import show
+from ..structures import extract_properties
 from ._metatomic import metatomic_featurizer
 
 
-__all__ = [
-    "explore",
-    "metatomic_featurizer",
-]
+__all__ = ["explore", "metatomic_featurizer", "get_featurizer"]
+
+FEATURIZER_VERSION_MAP = {"pet-mad-1.0": "1.0.0"}
+
+
+def get_featurizer(name, device=None, batch_size=1):
```

Comment:
```suggestion
def get_featurizer(name):
```

This function is more of a convenience function that could know about a bunch of featurizers, for some of which these would not be valid parameters. I think at this point we can make a choice for the user and give them a good default option.